File: easy-solution/minesp3164/941.py
class Solution(object):
    def validMountainArray(self, arr):
        biggestCount = 0
        biggest = 0
        if(len(arr) < 3):
            return False
        for i in range(len(arr)):
            if(arr[i] > biggest):
                biggest = arr[i]
                biggestCount = i
        truth = True
        if(biggestCount == len(arr) - 1): return False
        for j in range(len(arr)-1):
            if(arr[i] == biggest): return False
            if (j < biggestCount):
                if(arr[j] < arr[j + 1]):
                    truth = True
                else: return False
            if (j >= biggestCount):
                if(arr[j] > arr[j + 1] ):
                    truth = True
                else: return False
            return truth


        """
        :type arr: List[int]
        :rtype: bool
        """

File: easy-solution/Antidote21/941.c

// retry 
bool validMountainArray(int* arr, int arrSize){
    int i = 0;
    while(i+1 < arrSize && arr[i] < arr[i+1]) // 올라가는 조건 
        i++;
    if(i==0 || i==arrSize-1) //ex) 0 1 2 3 4 5 / 5 4 3 2 1
        return false;
    while(i+1< arrSize && arr[i] > arr[i+1]) // 내려가는 조건 
        i++;
    return i == arrSize-1;
}
